pom.xml

```diff
@@ -2094,8 +2099,8 @@
             <classifier>tests</classifier>
```

Comment:
We may be able to remove the older version of the HTTP Client test dependencies, now that HTTP Client 5 has the separate named JAR that we can import instead.

Reply:
I am not sure how this should be updated. Please can you provide more details. Thank you

Reply:
For the HTTP Client dependencies that contain the `<classifier>tests</classifier>` elements, just remove those dependencies from the POM. Sorry for the confusion.

Reply:
thank you for clarifying!